tests/fv3_conf/compile_bsub.IN_wcoss_dell_p3

```diff
@@ -3,7 +3,7 @@
 #BSUB -oo out
 #BSUB -eo err
 #BSUB -J @[JBNME]
-#BSUB -W 00:45
+#BSUB -W 01:00:00
```

Comment:
Using #BSUB -W 00:50 allowed both compile 16 and 17 to complete--about 2800 secs each.

Reply:
Ok, let me make this change.